src/pages/submission-list/submission-list.ts

```diff
@@ -262,26 +273,41 @@ export class SubmissionListPage {
       // this.utils.stopLoader();
     })
   }
-
   viewEntityReports() {
+    this.showEntityActionsheet = false;
+    this.showActionsheet = false;
     const payload = {
       entityId: this.submissionList[0].entityId,
       observationId: this.submissionList[0].observationId
     }
     this.navCtrl.push(ObservationReportsPage, payload);
   }
-
-
-  actions(submissionId, action) {
+  viewEntityReportsWithScore() {
+    console.log('clicked reports with score');
+    this.showEntityActionsheet = false;
+    this.showActionsheet = false;
+    const payload = {
+      entityId: this.submissionList[0].entityId,
+      observationId: this.submissionList[0].observationId
+    }
+    this.navCtrl.push(ReportsWithScorePage, payload);
+  }
+  actions(submissionId, action, submission) {
     // this.dap.checkForSubmissionDoc(submissionId, action);
+    submission.showActionsheet = false;
+    this.showActionsheet = false;
     this.navCtrl.push(ObservationReportsPage, { submissionId: submissionId })
   }
+  actionsWithScore(submissionId, action, submission) {
+    // this.dap.checkForSubmissionDoc(submissionId, action);
```

Comment:
both actions and actionsWithScore have same functionality, Please use same function to implement both functionalities